EIPS/eip-1.md

```diff
@@ -252,6 +252,24 @@ Permitted Networking Specifications URLs must anchor to a specific commit, and s
 ^https://github.com/ethereum/devp2p/blob/[0-9a-f]{40}/.*$
 ```
 
+### Web Hypertext Application Technology Working Group (WHATWG)
```

Comment:
```suggestion
### WHATWG
```
I weakly prefer the abbreviation.

Reply:
We require authors to expand abbreviations, so I figured we should probably too :rofl: